release-notes.txt

```diff
@@ -1,3 +1,548 @@
+* cpu/stm32u5: Enable rtc support (#20521)
+* drivers/lsm6dsxx: refactoring Lsm6dsl into common driver Lsm6dsxx (#20170)
+* native64: Separate board for 64-bit native (#20335)
+* native: Linux/x86_64 support (#20315)
+
+CPU
+---
+
++ cpu/native: add gpio-mock (#20431)
++ cpu/native: Add support for periph_timer_query_freqs (#20306)
++ drivers/periph_gpio_ll: Implement API to switch direction (#20292)
++ stdio_dispatch: allow to select multiple stdio methods at the same
+  time (#19738)
+* cpu/esp8266: allow arbitrary SPI clocks (#20410)
+* cpu/native/periph/uart: uart_poweroff() closes the file descriptor +
+  extended baud-rate support on Linux (#20365)
+* cpu/native: Fix executable stack warning (#20307)
+* cpu/native: return error code on failed assertion (#20419)
+* cpu/stm32/periph/gpio: simplify condition code (#20479)
+* cpu/stm32/periph/usbdev_fs: avoid using ztimer when not needed (#20467)
+* cpu/stm32: Fix ifdefs in periph_gpio_ll (#20530)
+* cpu/stm32: remove redundant conditional compile block (#20495)
+* drivers/mtd_flashpage: allow to define AUX slot on flash (#18608)
+* sys/byteorder: clean up implementation (#20313)
+* sys/cpp_new_delete: always enable the module when C++ is used (#20348)
+* tests: exit `native` with error value on failure (#20445)
+* treewide: Rust updates (#20303)
+
+Device Drivers
+--------------
+
+* driver/lsm6dsxx: Allow generic LSM6DSXX driver (#20504)
+* drivers/w5100: make use of netdev_eui48_get() (#20304)
+
+Documentation
+-------------
+
++ build system: add list of features for documentation and sanity
+  checking (#20366)
++ doc: add graphic to vision page (#20498)
++ doc: Add guide for dev setup on Windows (#20426)
++ doc: add terminal program configuration (#20493)
++ doc: doxygen: add "Topics" to menus (#20443)
++ doxygen: added community processes (import from Wiki) (#20496)
++ doxygen: added vision.md imported from Wiki (#20491)
+* boards/feather-nrf52840*: doc cleanup (#20422)
+* boards/nucleo-c031c6: fix ADC and enhance doc (#20502)
+* doc/roadmap: Make working 6LBR a goal (#20351)
+* doc: Allow resizing of doxygen side-nav (#20511)
+
+Build System / Tooling
+----------------------
+
++ build sytem: add `--no-warn-rwx-segments` to linker (#20381)
++ dist/tools/mspdebug: build from source (#20345)
+* Avoid using egrep (#20522)
+* build system: Improve failure mode for FEAUTRES_REQUIRED_ANY (#20408)
+* dist/tools/dhcpv6-pd_ia: recognize Debian correctly (#20487)
+* Enable compile_and_test_for_board to skip if nothing changed (#19546)
+* tools/openocd: define SWD_EN signal for tigard (#20507)
+* tree wide: accomodate RISC-V toolchain update (#20380)
+
+Kconfig
+-------
+
+* Remove Kconfig dependency modelling (#20438)
+
+Examples
+--------
+
+* examples/filesystem: Update README and increase default stack size (#20439)
+* examples/psa_crypto: Fix modules to run ECDSA on SEs (#20461)
+* examples/wakaama: rename to lwm2m (#20327)
+* exmple > example in Cargo.toml comment (#20399)
```

Comment:
```suggestion
* examples/rust-hello-world: Fix typo in Cargo.toml (#20399)
```

We will not get this past the CI (code spell) with the typo in it, so I just changed the PR title now to match above (and to be consistent with our commit guides).